src/store/link.ts

```diff
@@ -0,0 +1,38 @@
+import { createSlice, PayloadAction, createSelector } from "@reduxjs/toolkit";
+import { getParticipants } from "./zoom";
+
+interface LinkedAccountsState {
+  byId: {
+    [id: string]: {
+      id: string;
+      discord: string;
+    };
+  };
+};
+
+const initialState: LinkedAccountsState = {
+  byId: {},
+};
+
+type UserLinkAction = PayloadAction<{ id: string, discord: string }>;
+
+const { reducer, actions, name } = createSlice({
+  name: "linkedAccounts",
+  reducers: {
+    accountLinked: (state, { payload }: UserLinkAction) => {
+      state.byId[payload.id] = payload;
+    }
+  },
+  initialState,
+});
+
+type RootState = { [name]: LinkedAccountsState };
```

Comment:
Import `RootState` from `store/index.ts` instead. (This should be OK, because circular dependencies are fine for types--If it's not, then ignore this entire comment.)